packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -2035,18 +2432,18 @@ describe("AttributesController", function () {
             expect(result0).toStrictEqual([]);
 
             const result1 = await consumptionController.attributes.getPredecessorsOfAttribute(repositoryAttributeVersion1.id);
-            expect(result1).toStrictEqual([repositoryAttributeVersion0]);
+            expect(JSON.stringify(result1)).toStrictEqual(JSON.stringify([repositoryAttributeVersion0]));
```

Comment:
Why was it necessary to add these `JSON.stringify(...)` calls?

Reply:
Yeah, it's kind of weird behavior. So, if I don't add the `JSON.stringify(...)` calls, I'll get an error like the following: 

![image](https://github.com/user-attachments/assets/cbbbd782-c16d-4d7a-bd92-d2d6470b0181)

I think it is because for one side of the comparison `default` is explicitly set to `undefined`, since it was true before, but for the other it isn't listed explicitly, since it is undefined. I found this as a work around.